source/utilities.tex

```diff
@@ -16301,6 +16310,68 @@
 core constant expressions\iref{expr.const}.
 \end{itemdescr}
 
+\indexlibrarymember{check_dynamic_spec}{basic_format_parse_context}%
+\begin{itemdecl}
+template<class... Ts>
+  constexpr void check_dynamic_spec(size_t id) noexcept;
+\end{itemdecl}
+
+\begin{itemdescr}
+\pnum
+\mandates
+The types in \tcode{Ts...} are unique.
+Each type in \tcode{Ts...} is one of
+\keyword{bool},
+\tcode{char_type},
+\keyword{int},
+\keyword{unsigned} \keyword{int},
+\keyword{long} \keyword{long} \keyword{int},
+\keyword{unsigned} \keyword{long} \keyword{long} \keyword{int},
+\keyword{float},
+\keyword{double},
+\keyword{long} \keyword{double},
```

Comment:
```suggestion
\tcode{\keyword{unsigned} \keyword{int}},
\tcode{\keyword{long} \keyword{long} \keyword{int}},
\tcode{\keyword{unsigned} \keyword{long} \keyword{long} \keyword{int}},
\keyword{float},
\keyword{double},
\tcode{\keyword{long} \keyword{double}},
```

Reply:
done, but why bother? because the spacing is slightly different?

Reply:
It better encodes what is meant.